app/assets/javascripts/reducers/columns/done.js

```diff
@@ -33,7 +34,18 @@ const done = (state = initialState, action) => {
         stories: [],
       }));
     
-      return { sprints };
+      return { ...state, sprints };
+    case actionTypes.TOGGLE_STORY:
+      return {
+        ...state,
+        sprints: (state.sprints.map((sprint) => {
+          return {
+            ...sprint,
+            stories: collapseStory(sprint.stories, action.id)
+          }
+        })
+        )
+      };
```

Comment:
wouldnt be better to fomat like 
           })
        )};
??

Reply:
this `)` is to separate the map method in line 41 just for better understanding.